services/cilium-hubble-relay-traefik/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+type: internal
```

Comment:
Its `internal` so not really matters ig but lets add 
```
requiredDependencies
- traefik
```
or atleast 
```
dependencies:
  - traefik
```
if we want to keep things loosely coupled.

We can also add 
```
allowMultipleInstances: false
```
for future consumption.

Reply:
Thanks!

Reply:
Since the app creates a resource defined by a traefik CRD, the `requiredDependencies` makessense.

Reply:
Fixed in https://github.com/mesosphere/kommander-applications/pull/3289/commits/a16cf884069f9ba4bd619e1feb52f5ef5a902440